apps/total-typescript/src/components/primary-newsletter-cta.tsx

```diff
@@ -88,12 +97,61 @@ export const ArticleNewsletterCta = () => {
         Stay up-to-date on the latest news and updates from the world of
         TypeScript.
       </p>
-      <SubscribeToConvertkitForm
-        actionLabel="Subscribe"
-        onSuccess={(subscriber?: any, email?: string) => {
-          return handleOnSuccess(router, subscriber, email)
-        }}
-      />
+      <div className="relative flex items-center justify-center">
+        <SubscribeToConvertkitForm
+          actionLabel="Subscribe"
+          fields={readArticleField}
```

Comment:
adding a custom subscribed field `read_writing_string_replace_in_typescript_on: 2022-09-02` so that we can tell how this CTA performs on each article.